openedx_events/learning/data.py

```diff
@@ -190,3 +190,29 @@ class CourseDiscussionConfigurationData:
     unit_level_visibility = attr.ib(type=bool, default=False)
     plugin_configuration = attr.ib(type=dict, default={})
     contexts = attr.ib(type=List[DiscussionTopicContext], factory=list)
+
+
+@attr.s(frozen=True)
+class PersistentCourseGradeData:
+    """
+    Attributes defined for Open edX PersistentCourseGrade data object.
+
+    Arguments:
+        user_id (int): identifier of the grade to which the grade belongs.
```

Comment:
```suggestion
        user_id (int): identifier of the user to which the grade belongs.
```

Reply:
Oh yes, that's right thanks a lot, Carlos, fixed.